lib/cldr/export/data/country_codes.rb

```diff
@@ -0,0 +1,28 @@
+module Cldr
+  module Export
+    module Data
+      class CountryCodes < Base
+        def initialize
+          super(nil)
+          update(:CountryCodes => country_codes)
```

Comment:
```suggestion
          update(:country_codes => country_codes)
```
For consistency with the file name.